tank/README.md

```diff
@@ -0,0 +1,13 @@
+To run the fine tuning example, from the root SHARK directory, run:
+
+```shell
+IMPORTER=1 ./setup_venv.sh
+source shark.venv/bin/activate
+pip install jupyter
```

Comment:
maybe add the `tf-models-nightly` here

Reply:
makes sense since this is the only thing that requires it. I'll add a try/catch on the import in case someone misses the instructions too with an error that provides a copy pastable pip install command